src/OsmMom.sol

```diff
@@ -75,23 +71,39 @@ contract OsmMom {
 
     mapping (bytes32 => address) public osms;
 
-    constructor() public {
+    constructor(address vat_) {
+        vat = vat_;
         owner = msg.sender;
+        emit SetOwner(msg.sender);
+    }
+
+    function file(bytes32 what, address data) external onlyOwner {
```

Comment:
Consider also using a `file` function to set `osms` as there are now 2 ways to configure things (assuming setOwner and setAuthority can remain separate as we want to preserve ds-auth interface). This suggestions is a bit irritating as will require an execlib update, so fine either way.

Reply:
Yeah I have some mixed feelings here. If it would be a complete new module I wouldn't doubt to use the `file` convention, but taking into account this is an upgrade would prefer to be less disruptive and not have to change the `dss-exec-lib`.